test/gurka/test_instructions_roundabout.cc

```diff
@@ -66,7 +66,7 @@ TEST_F(InstructionsRoundabout, RoundaboutEnterOnly) {
   gurka::assert::raw::expect_maneuvers(result, {DirectionsLeg_Maneuver_Type_kStart,
                                                 DirectionsLeg_Maneuver_Type_kRoundaboutEnter,
                                                 DirectionsLeg_Maneuver_Type_kDestination});
-  int maneuver_index = 1;
+  [[maybe_unused]] int maneuver_index = 1;
```

Comment:
Instead of deleting using `[[maybe_unused]]`

Reply:
are they used at all? or just in some ifdef'd block?